src/EnergyPlus/Furnaces.cc

```diff
@@ -12636,7 +12636,7 @@ namespace Furnaces {
             state.dataFurnaces->Furnace(FurnaceNum).MinOATCompressorCooling =
                 DXCoils::GetMinOATCompressorUsingIndex(state, CoolingCoilIndex, errFlag);
         } else if (state.dataFurnaces->Furnace(FurnaceNum).CoolingCoilType_Num == Coil_CoolingAirToAirVariableSpeed) {
-            state.dataFurnaces->Furnace(FurnaceNum).MinOATCompressorHeating =
+            state.dataFurnaces->Furnace(FurnaceNum).MinOATCompressorCooling =
```

Comment:
I don't mean to blow up this PR (in terms of changes), but `auto &thisFurnace = state.dataFurnaces->Furnace(FurnaceNum);` would clean up the code somewhat.

Reply:
Thanks! I will do some small refactoring based on your comments. Then probably leave you guys to comment on it. @amirroth @rraustad

Reply:
@jcyuan2020 don't worry about it, I just made the changes.

Reply:
Thanks! @rraustad